Gemfile

```diff
@@ -76,6 +76,9 @@ gem 'yard'
 # It should not be updated without verifying that the behavior still matches JS version 4.4.2.
 gem 'zxcvbn', '0.1.7'
 
+# Lookbook must be listed after ViewComponent, so listed here out of default alphabetical order.
+gem 'lookbook', '~> 1.2.1', require: false
```

Comment:
Now that we're doing `require: false` there's no way this needs to be after, I bet we could alphabetize now

Reply:
> Now that we're doing `require: false` there's no way this needs to be after, I bet we could alphabetize now

👍 Good call. Updated in c8c3194.